src/styles/global.css

```diff
@@ -0,0 +1,14 @@
+/* Borrowing the global.css concept from https://www.gatsbyjs.com/docs/recipes/styling-css/ */
+
+html {
+  /* background-color: lavenderblush; */
+}
+h1 {
+  display: flex;
+  padding: 5px;
+  font-size: 180%;
+  line-height: 64px;
+  text-align: center;
+  vertical-align: middle;
+  gap: 0px 5px;
+}
```

Comment:
There is a problem with this approach: It uses a fixed font size at all screen sizes. Also, since only `h1` was modified, there is now a mismatch betweeen `h1` and `h2`. For example, see the difference between `About Us` and `Founders` on https://marmaladeai-fixnichepagetitlewrappingonpho.gtsb.io/about

Compare that to my approach in #172 in `src/css/tailwind.css` (https://github.com/Marmalade-ai/company-website/pull/172/files), which preserves the relative sizes for h1, h2, h3 while also adjust the font size for the screen size.